### ISSUE TITLE
OpenSSL failed to nmake clean due to fatal error U1095 on windows with MSVC

### ISSUE THREAD

**Issue by spacelg** [openssl/openssl#10553]
Environment:
Windows Server 2016 + VS2017 + openssl master branch latest source code.

Issue description:
I tried to build openssl using VS2017 on Windows Server 2016. We found that openssl failed to nmake clean due to fatal error U1095: expanded command line 'rmdir /Q /S doc\html\man3\ADMISSIONS.html  on windows with MSVC. It can be reproduced on latest master revision be3acd7 reversion. Could you please help take a look at this? Thanks in advance!

Repro steps:
1. git clone https://github.com/openssl/openssl D:\openssl\src
2. cd D:\openssl\src
3. perl Configure VC-WIN32
4. nmake clean

[log_x86_build_1.log](https://github.com/openssl/openssl/files/3909253/log_x86_build_1.log)

Actual result:
NMAKE : fatal error U1095: expanded command line 'rmdir /Q /S doc\html\man3\ADMISSIONS.html

**Comment by mspncp** [comment]
I get the same error [U1095](https://docs.microsoft.com/en-us/cpp/error-messages/tool-errors/nmake-fatal-error-u1095?view=vs-2019) when I apply exactly the same steps.

```
expanded command line 'commandline' too long

After macro expansion, the given command line exceeded the limit on length of command lines for the operating system.
```

The excessively long command tries to remove all html files explicitly using an **rmdir** command?!

Looks like a bug @levitte.

```
	rmdir /Q /S doc\html\man3\ADMISSIONS.html doc\html\man3\ASN1_INTEGER_get_int64.html doc\html\man3\ASN1_ITEM_lookup.html doc\html\man3\ASN1_OBJECT_new.html doc\html\man3\ASN1_STRING_TABLE_add.html doc\html\man3\ASN1_STRING_length.html doc\html\man3\ASN1_STRING_new.html doc\html\man3\ASN1_STRING_print_ex.html
	...
```

**Comment by mspncp** [comment]
Side note: @spacelg do you really intend to build **32bit** binaries for Windows Server 2016? For the 64bit binaries, the correct configure target VC-WIN64A.

**Comment by mspncp** [comment]
@levitte a git bisection identifies b0940b33a60cbfab03153287181372d0066410a6 as the offending commit.

Note also that the Configure command runs significantly longer than usual in the bad case.

**Comment by mspncp** [comment]
Oh, and I forgot to mention: thank you @spacelg for taking the time to report this bug!

**Comment by levitte** [comment]
Fix submitted, please try it out: #10555
